keyboards/slothie60/keymaps/60_iso/keymap.c

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	//main layer
+	[0] = LAYOUT_60_iso(
+        QK_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL
+	),
+	//fn1-layer
+	[1] = LAYOUT_60_iso(
+        QK_BOOT, KC_F1,	  KC_F2, 	KC_F3,    KC_F4,     KC_F5,     KC_F6,     KC_F7,     KC_F8,     KC_F9,      KC_F10,  KC_F11,   KC_F12,  KC_BSPC,
+		_______, _______, KC_UP,    _______,  _______,   _______,   _______,   _______,   _______,   _______,    _______, _______,  _______,
+        _______, KC_LEFT, KC_DOWN,  KC_RIGHT, _______,   _______,   _______,   _______,   _______,   _______,    _______, _______,  _______, _______,
+        _______, _______, _______,  _______,  _______,   _______,   _______,   _______,   _______,   _______,    _______, _______,  _______,
+        _______, _______, _______,                                  _______,                          			 _______, _______,  _______,  _______
+    )
```

Comment:
This is a dead layer
```suggestion
	)
```

Reply:
What do you mean by dead layer? I'd like to have the fn-keys and numpad on layer 1, was this not the way to do it?

Reply:
You don't have `MO(1)` or anything else like that on the layer 0 in this keymap, therefore layer 1 is unreachable.  You need to find some place for `MO(1)` in keymaps with non-split right Shift.

Reply:
God, i'm an idiot. Thanks for pointing that out. I'll make the changes.